src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallLibertyTask.groovy

```diff
@@ -24,18 +24,20 @@ class InstallLibertyTask extends AbstractTask {
 
     @TaskAction
     void install() {
-        def params = buildInstallLibertyMap(project)
-
-        project.ant.taskdef(name: 'installLiberty',
-                            classname: 'net.wasdev.wlp.ant.install.InstallLibertyTask',
-                            classpath: project.buildscript.configurations.classpath.asPath)
-        project.ant.installLiberty(params)
-
-        String licenseFilePath = project.configurations.getByName('libertyLicense').getAsPath()
-        if (licenseFilePath) {
-            def command = "java -jar " + licenseFilePath + " --acceptLicense " + project.buildDir
-            def process = command.execute()
-            process.waitFor()
+        if (!isLibertyInstalled(project)) {
+            def params = buildInstallLibertyMap(project)
+
+            project.ant.taskdef(name: 'installLiberty',
+                                classname: 'net.wasdev.wlp.ant.install.InstallLibertyTask',
+                                classpath: project.buildscript.configurations.classpath.asPath)
+            project.ant.installLiberty(params)
+
+            String licenseFilePath = project.configurations.getByName('libertyLicense').getAsPath()
+            if (licenseFilePath) {
+                def command = "java -jar " + licenseFilePath + " --acceptLicense " + project.buildDir
+                def process = command.execute()
+                process.waitFor()
+            }
         }
```

Comment:
I think the else of this should say something.  If we called to the ant task, it would say "Liberty profile is already installed."
Maybe we should log.info 
"Liberty is already installed at " + getInstallDir(project)